doc/TechnicalOverview.md

```diff
@@ -149,8 +149,7 @@ spikes are transferred from one end to another. Both ends, publishers and
 subscribers, connect through a given URL schema.  In the case of spike
 streaming, the URL schema is "monsteer://". If auto-dIscovery is enabled giving
 this schema is sufficient for both ends to connect, otherwise it is also
-possible to hardcode the host and
-its port for the connection.
+possible to hardcode the host and its port for the connection.
```

Comment:
The whole section needs to be rewritten (if not the whole documentation). One detail that I don't know how to clarify is that schemes are used only to enable the Brion plugins but now they are not used by ZeroEQ at all. I need to understand this whole change myself.